diff --git themes/bartik/bartik.info themes/bartik/bartik.info
new file mode 100644
index 0000000..5f4a973
--- /dev/null
+++ themes/bartik/bartik.info
@@ -0,0 +1,39 @@
+; $Id$
+
+name = Bartik
+description = A flexible, recolorable theme with many regions.
+package = Core
+version = VERSION
+core = 7.x
+engine = phptemplate
+
+stylesheets[all][] = css/layout.css
+stylesheets[all][] = css/style.css
+stylesheets[all][] = css/colors.css
+stylesheets[print][] = css/print.css
+stylesheets[all][] = css/maintenance-page.css
+
+scripts[] = scripts/search.js
+
+regions[header] = Header
+regions[help] = Help
+regions[page_top] = Page top
+regions[page_bottom] = Page bottom
+regions[highlight] = Highlighted
+
+regions[featured] = Featured
+regions[content] = Content
+regions[sidebar_first] = Sidebar first
+regions[sidebar_second] = Sidebar second
+
+regions[triptych_first] = Triptych first
+regions[triptych_middle] = Triptych middle
+regions[triptych_last] = Triptych last
+
+regions[footer_firstcolumn] = Footer first column
+regions[footer_secondcolumn] = Footer second column
+regions[footer_thirdcolumn] = Footer third column
+regions[footer_fourthcolumn] = Footer fourth column
+regions[footer] = Footer
+
+settings[shortcut_module_link] = 1
diff --git themes/bartik/color/color.inc themes/bartik/color/color.inc
new file mode 100644
index 0000000..dff86bf
--- /dev/null
+++ themes/bartik/color/color.inc
@@ -0,0 +1,115 @@
+<?php
+// $Id$
+
+// Put the logo path into JavaScript for the Preview JS.
+drupal_add_js(array('color' => array('logo' => theme_get_setting('logo', 'bartik'))), 'setting');
+
+$info = array(
+  // Available colors and color labels used in theme.
+  'fields' => array(
+    'bg' => t('Main background'),
+    'link' => t('Link color'),
+    'top' => t('Header top'),
+    'bottom' => t('Header bottom'),
+    'text' => t('Text color'),
+    'sidebar' => t('Sidebar background'),
+    'sidebarborders' => t('Sidebar borders'),
+    'footer' => t('Footer background'),
+  ),
+  // Pre-defined color schemes.
+  'schemes' => array(
+    'default' => array(
+      'title' => t('Bartik (Default)'),
+      'colors' => array(
+        'bg' => '#ffffff',
+        'link' => '#0073b6',
+        'top' => '#000000',
+        'bottom' => '#3d3d3d',
+        'text' => '#3b3b3b',
+        'sidebar' => '#feffff',
+        'sidebarborders' => '#d0d0d0',
+        'footer' => '#161617',
+      ),
+    ),
+    'Blue Lagoon' => array(
+      'title' => t('Blue Lagoon'),
+      'colors' => array(
+        'bg' => '#ffffff',
+        'link' => '#0073b6',
+        'top' => '#2385c2',
+        'bottom' => '#5ab5ee',
+        'text' => '#3b3b3b',
+        'sidebar' => '#feffff',
+        'sidebarborders' => '#4377ac',
+        'footer' => '#00143d',
+      ),
+    ),
+    'Plum' => array(
+      'title' => t('Plum'),
+      'colors' => array(
+        'bg' => '#f6eecc',
+        'link' => '#6b2454',
+        'top' => '#461453',
+        'bottom' => '#6b2454',
+        'text' => '#2c233f',
+        'sidebar' => '#f0f0f0',
+        'sidebarborders' => '#4e2e44',
+        'footer' => '#452632',
+      ),
+    ),
+    'Fresh' => array(
+      'title' => t('Fresh'),
+      'colors' => array(
+        'bg' => '#f6ffff',
+        'link' => '#087095',
+        'top' => '#78b91a',
+        'bottom' => '#5b8b15',
+        'text' => '#585858',
+        'sidebar' => '#feffff',
+        'sidebarborders' => '#5899b0',
+        'footer' => '#2f4016',
+      ),
+    ),
+  ),
+
+  // CSS files (excluding @import) to rewrite with new color scheme.
+  'css' => array(
+    'css/colors.css',
+  ),
+
+  // Files to copy.
+  'copy' => array(
+    'logo.png',
+  ),
+
+  // Gradient definitions.
+  'gradients' => array(
+    array(
+      // (x, y, width, height).
+      'dimension' => array(0, 0, 0, 0),
+      // Direction of gradient ('vertical' or 'horizontal').
+      'direction' => 'vertical',
+      // Keys of colors to use for the gradient.
+      'colors' => array('top', 'bottom'),
+    ),
+  ),
+
+  // Color areas to fill (x, y, width, height).
+  'fill' => array(),
+
+  // Coordinates of all the theme slices (x, y, width, height)
+  // with their filename as used in the stylesheet.
+  'slices' => array(),
+
+  // Reference color used for blending. Matches the base.png's colors.
+  'blend_target' => '#ffffff',
+
+  // Preview files.
+  'preview_image' => 'color/preview.png',
+  'preview_css' => 'color/preview.css',
+  'preview_js' => 'color/preview.js',
+  'preview_html' => 'color/preview.html',
+
+  // Base file for image generation.
+  'base_image' => 'color/base.png',
+);
diff --git themes/bartik/color/preview.css themes/bartik/color/preview.css
new file mode 100644
index 0000000..9a5a557
--- /dev/null
+++ themes/bartik/color/preview.css
@@ -0,0 +1,68 @@
+/* $Id$ */
+
+/* Bring in the rest of the theme's CSS styles. */
+@import url("../css/style.css");
+
+/**
+ * From color.css
+ */
+#preview, #preview-header {
+  background-color: #000000;
+}
+
+/**
+ * Undoing Seven's reset.
+ */
+#preview #preview-header #preview-logo img {
+  border: 0;
+}
+
+/**
+ * Basic layout styles
+ */
+#preview {
+  width: 640px;
+  overflow: hidden;
+  font-size: 1em;
+}
+#preview #preview-header div.section,
+#preview #preview-navigation div.section,
+#preview #preview-featured div.section,
+#preview #preview-main {
+  width: 960px;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0 20px;
+}
+#preview #preview-content,
+#preview .sidebar {
+  display: inline;
+  float: left; /* LTR */
+  position: relative;
+}
+#preview #preview-content {
+  width: 720px;
+}
+#preview .sidebar {
+  width: 240px;
+}
+#preview #preview-content .section,
+.sidebar .section {
+  padding: 0 10px;
+}
+#preview #footer-wrapper {
+  padding: 35px 20px 30px;
+}
+#preview #footer-firstcolumn,
+#preview #footer-secondcolumn,
+#preview #footer-thirdcolumn,
+#preview #footer-fourthcolumn {
+  padding: 0 10px;
+  width: 220px;
+  display: inline;
+  float: left; /* LTR */
+  position: relative;
+}
+#preview #footer {
+  width: 940px;
+}
diff --git themes/bartik/color/preview.html themes/bartik/color/preview.html
new file mode 100644
index 0000000..13537ed
--- /dev/null
+++ themes/bartik/color/preview.html
@@ -0,0 +1,105 @@
+<div id="preview">
+  <div id="preview-header">
+    <div class="section clearfix">
+      <a href="/" title="Home" rel="home" id="preview-logo" name="logo"><img src="../../../themes/bartik/logo.png" alt="Home" /></a>
+      <div id="preview-name-and-slogan">
+        <div id="preview-site-name">
+          <strong><a href="#" title="Home" rel="home"><span>Bartik</span></a></strong>
+        </div>
+      </div><!-- /#name-and-slogan -->
+    </div>
+  </div><!-- /.section, /#header -->
+  <div id="preview-main-wrapper">
+    <div id="preview-main" class="clearfix">
+      <div id="preview-sidebar-first" class="column sidebar">
+        <div class="section">
+          <div class="region region-sidebar-first">
+            <div id="preview-block-user-login" class="block block-user first last even">
+              <h2>
+                Etiam est risus
+              </h2>
+              <div class="content">
+                <p>
+                  Maecenas id porttitor felis. Pellentesque mollis urna in nibh pharetra semper. Nulla erat odio, imperdiet quis cursus vitae, ultricies
+                  at diam.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div><!-- /.section, /#sidebar-first -->
+      <div id="preview-content" class="column">
+        <div class="section">
+          <a id="preview-main-content" name="main-content"></a>
+          <h1 class="title" id="preview-page-title">
+            Lorem ipsum dolor
+          </h1>
+          <div class="region region-content">
+            <div id="preview-block-system-main" class="block block-system first last even">
+              <div class="content">
+                <div id="preview-node-1" class="node node-page clearfix" about="/node/1" typeof="foaf:Document">
+                  <div class="content clearfix">
+                    <div class="field field-name-body field-type-text-with-summary field-label-hidden clearfix">
+                      <div class="field-items">
+                        <div class="field-item even" property="content:encoded">
+                          <p>
+                            Sit amet, <a href="#">consectetur adipisicing elit</a>, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+                            veniam, quis nostrud <a href="#">exercitation ullamco</a> laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
+                            in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+                            sunt in culpa qui officia deserunt mollit anim id est laborum.
+                          </p>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div><!-- /.section, /#content -->
+    </div>
+  </div><!-- /#main, /#main-wrapper -->
+
+  <div id="footer-wrapper"><div class="section">
+      <div id="footer-columns" class="clearfix">
+          <div id="footer-firstcolumn" class="region sitemap"><div class="section">
+            <ul>
+              <li><a>Donec placerat</a></li>
+              <li><a>Nullam nibh dolor</a></li>
+              <li><a>Blandit sed</a></li>
+              <li><a>Fermentum id</a></li>
+            </ul>
+          </div></div> <!-- /.section, /#footer-firstcolumn -->
+          <div id="footer-secondcolumn" class="region sitemap"><div class="section">
+            <ul>
+              <li><a>Imperdiet sit amet</a></li>
+              <li><a>Nam mollis</a></li>
+              <li><a>Ultrices justo</a></li>
+              <li><a>Sed tempor</a></li>
+              </ul>
+            <ul>
+          </div></div> <!-- /.section, /#footer-secondcolumn -->
+          <div id="footer-thirdcolumn" class="region sitemap"><div class="section">
+           <ul>
+              <li><a>Sit amet</a></li>
+              <li><a>Gravida eget</a></li>
+              <li><a>Porta at</a></li>
+              <li><a>Nam non</a></li>
+            </ul>
+          </div></div> <!-- /.section, /#footer-thirdcolumn -->
+          <div id="footer-fourthcolumn" class="region sitemap"><div class="section">
+            <ul>
+              <li><a>Sed vitae</a></li>
+              <li><a>Tellus</a></li>
+              <li><a>Etiam sem</a></li>
+              <li><a>Arcu eleifend</a></li>
+            </ul>
+          </div></div> <!-- /.section, /#footer-fourthcolumn -->
+      </div><!-- /#footer-columns -->
+      <div id="footer" class="clearfix">
+        Aliquam aliquet, est <a>a ullamcorper</a> condimentum.
+    </div><!-- /#footer -->
+  </div></div> <!-- /.section, /#footer-wrapper -->
+
+</div><!-- /#preview -->
diff --git themes/bartik/color/preview.js themes/bartik/color/preview.js
new file mode 100644
index 0000000..19f99ce
--- /dev/null
+++ themes/bartik/color/preview.js
@@ -0,0 +1,35 @@
+/* $Id$ */
+
+(function ($) {
+  Drupal.color = {
+    logoChanged: false,
+    callback: function(context, settings, form, farb, height, width) {
+      // Change the logo to be the real one.
+      if (!this.logoChanged) {
+        $('#preview #preview-logo img').attr('src', Drupal.settings.color.logo);
+        this.logoChanged = true;
+      }
+
+      // Solid background.
+      $('#preview', form).css('backgroundColor', $('#palette input[name="palette[bg]"]', form).val());
+
+      // Text preview
+      $('#preview #preview-main h2, #preview #preview-main p', form).css('color', $('#palette input[name="palette[text]"]', form).val());
+      $('#preview #preview-content a', form).css('color', $('#palette input[name="palette[link]"]', form).val());
+
+      // Sidebar background
+      $('#preview .sidebar .block', form).css('background-color', $('#palette input[name="palette[sidebar]"]', form).val());
+
+      // Footer background
+      $('#preview #footer-wrapper', form).css('background-color', $('#palette input[name="palette[footer]"]', form).val());
+
+      $('#preview .sidebar .block', form).css('border-color', $('#palette input[name="palette[sidebarborders]"]', form).val());
+
+      // CSS3 Gradients
+      var gradient_start = $('#palette input[name="palette[top]"]', form).val();
+      var gradient_end = $('#palette input[name="palette[bottom]"]', form).val();
+
+      $('#preview #preview-header', form).attr('style', "background-color: " + gradient_start + "; background-image: -webkit-gradient(linear, 0% 0%, 0% 100%, from(" + gradient_start + "), to(" + gradient_end + ")); background-image: -moz-linear-gradient(-90deg, " + gradient_start + ", " + gradient_end + ");");
+    }
+  };
+})(jQuery);
diff --git themes/bartik/css/colors.css themes/bartik/css/colors.css
new file mode 100644
index 0000000..8879512
--- /dev/null
+++ themes/bartik/css/colors.css
@@ -0,0 +1,36 @@
+/* $Id$ */
+
+/**
+ * Styles and colors that the color module modifies.
+ */
+body {
+  background-color: #161617;
+  color: #ffffff;
+}
+#page-wrapper,
+body.overlay {
+  background-color: #ffffff;
+  color: #3b3b3b;
+}
+#header {
+  background-color: #000000;
+  background-image: -webkit-gradient(linear, 0% 0%, 0% 100%, from(#000000), to(#3d3d3d));
+  background-image: -moz-linear-gradient(-90deg, #000000, #3d3d3d);
+}
+a {
+  color: #0073b6;
+}
+a:hover,
+a:focus {
+  color: #018fe2;
+}
+a:active {
+  color: #23aeff;
+}
+.sidebar .block {
+  background-color: #feffff;
+  border-color: #d0d0d0;
+}
+#footer-wrapper {
+  background: #161617;
+}
diff --git themes/bartik/css/ie-rtl.css themes/bartik/css/ie-rtl.css
new file mode 100644
index 0000000..6b5d435
--- /dev/null
+++ themes/bartik/css/ie-rtl.css
@@ -0,0 +1,11 @@
+/* $Id: ie-rtl.css,v 1.2 2010/05/18 02:32:53 jensimmons Exp $ */
+
+fieldset legend {
+  left: 6px;
+}
+.tabs ul.primary li a.active {
+  padding: 0 10px 0 7px;
+}
+ul.action-links li a {
+  zoom: 1;
+}
diff --git themes/bartik/css/ie.css themes/bartik/css/ie.css
new file mode 100644
index 0000000..4c8003a
--- /dev/null
+++ themes/bartik/css/ie.css
@@ -0,0 +1,24 @@
+/* $Id$ */
+
+.block {
+  zoom: 1;
+}
+input.form-text {
+  font-family: sans-serif;
+}
+#password-strength-text {
+  margin-top: 0;
+}
+fieldset legend {
+  left: -8px;
+  padding: 0;
+}
+.tabs ul.primary {
+  height: auto;
+}
+.tabs ul.primary li a.active {
+  padding: 0 7px 0 10px; /* LTR */
+}
+#footer-wrapper #footer .block {
+  height: 100%;
+}
diff --git themes/bartik/css/ie6.css themes/bartik/css/ie6.css
new file mode 100644
index 0000000..f49745a
--- /dev/null
+++ themes/bartik/css/ie6.css
@@ -0,0 +1,11 @@
+/* $Id$ */
+
+#content {
+  overflow: hidden;
+}
+.form-item-search-block-form {
+  width: 50%;
+}
+.tabs ul.primary {
+  zoom: 1;
+}
diff --git themes/bartik/css/layout-rtl.css themes/bartik/css/layout-rtl.css
new file mode 100644
index 0000000..d03ab86
--- /dev/null
+++ themes/bartik/css/layout-rtl.css
@@ -0,0 +1,16 @@
+/* $Id$ */
+
+/**
+ * Basic layout styles RTL overrides
+ */
+#content,
+.sidebar,
+#triptych-first,
+#triptych-middle,
+#triptych-last,
+#footer-firstcolumn,
+#footer-secondcolumn,
+#footer-thirdcolumn,
+#footer-fourthcolumn {
+  float: right;
+}
diff --git themes/bartik/css/layout.css themes/bartik/css/layout.css
new file mode 100644
index 0000000..d7b2b87
--- /dev/null
+++ themes/bartik/css/layout.css
@@ -0,0 +1,68 @@
+/* $Id$ */
+
+/**
+ * Basic layout styles
+ */
+#header div.section,
+#navigation div.section,
+#featured div.section,
+#messages,
+#main,
+#triptych,
+#footer-columns,
+#footer  {
+  width: 960px;
+  margin-left: auto;
+  margin-right: auto;
+}
+#content,
+.sidebar,
+#triptych-first,
+#triptych-middle,
+#triptych-last,
+#footer-firstcolumn,
+#footer-secondcolumn,
+#footer-thirdcolumn,
+#footer-fourthcolumn {
+  display: inline;
+  float: left; /* LTR */
+  position: relative;
+}
+.one-sidebar #content {
+  width: 720px;
+}
+.two-sidebars #content {
+  width: 480px;
+}
+.sidebar {
+  width: 240px;
+}
+
+#messages div.section,
+#content .section,
+.sidebar .section {
+  padding: 0 15px;
+}
+#breadcrumb {
+  margin: 0 15px;
+}
+#triptych-first,
+#triptych-middle,
+#triptych-last {
+  margin: 20px 20px 30px;
+  width: 280px;
+}
+#footer-wrapper {
+  padding: 35px 5px 30px;
+}
+#footer-firstcolumn,
+#footer-secondcolumn,
+#footer-thirdcolumn,
+#footer-fourthcolumn {
+  padding: 0 10px;
+  width: 220px;
+}
+#footer {
+  width: 940px;
+  min-width: 920px;
+}
diff --git themes/bartik/css/maintenance-page.css themes/bartik/css/maintenance-page.css
new file mode 100644
index 0000000..90e52eb
--- /dev/null
+++ themes/bartik/css/maintenance-page.css
@@ -0,0 +1,43 @@
+/* $Id$ */
+
+body.maintenance-page  {
+  background-color: #fff;
+  color: #000;
+}
+.maintenance-page #page-wrapper {
+  margin-left: auto;
+  margin-right: auto;
+  width: 800px;
+  border: 1px solid #666;
+  margin-top: 40px;
+}
+.maintenance-page #page {
+  margin: 20px 40px 50px;
+}
+.maintenance-page #header,
+.maintenance-page #messages,
+.maintenance-page #main {
+  width: auto;
+}
+.maintenance-page #header {
+  background-color: #fff;
+  background-image: none;
+}
+.maintenance-page #header,
+.maintenance-page #header a,
+.maintenance-page #header a:hover,
+.maintenance-page #header a:hover {
+  color: #000;
+}
+.maintenance-page #main {
+  margin: 0;
+}
+.maintenance-page  h1#page-title {
+  line-height: 1em;
+}
+.maintenance-page #messages {
+  padding: 0;
+}
+.maintenance-page #footer-wrapper {
+  display: none;
+}
diff --git themes/bartik/css/print.css themes/bartik/css/print.css
new file mode 100644
index 0000000..6e85314
--- /dev/null
+++ themes/bartik/css/print.css
@@ -0,0 +1,47 @@
+/* $Id$ */
+
+/* ---------- General Layout ---------- */
+
+body,
+input,
+textarea,
+select {
+  color: #000;
+  background: none;
+}
+body.two-sidebars,
+body.sidebar-first,
+body.sidebar-second,
+body {
+  width: 640px;
+}
+#sidebar-first,
+#sidebar-second,
+#navigation,
+#toolbar,
+#footer-wrapper,
+.tabs,
+.add-or-remove-shortcuts {
+  display: none;
+}
+.one-sidebar #content,
+.two-sidebars #content {
+  width: 100%;
+}
+#triptych-wrapper {
+  width: 960px;
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+#triptych-first, #triptych-middle, #triptych-last {
+  width: 250px;
+}
+
+/* ---------- Node Pages ---------- */
+
+#comments .title,
+#comments form,
+.comment_forbidden {
+  display: none;
+}
diff --git themes/bartik/css/style-rtl.css themes/bartik/css/style-rtl.css
new file mode 100644
index 0000000..67b0548
--- /dev/null
+++ themes/bartik/css/style-rtl.css
@@ -0,0 +1,236 @@
+/* $Id$ */
+
+/* ------------------ Reset Styles ------------------ */
+
+caption,
+th,
+td {
+  text-align: right;
+}
+blockquote{
+  border-left: none;
+  border-right: 4px solid #afafaf;
+}
+blockquote:before {
+  content: "\201D";
+}
+blockquote:after {
+  content: "\201C";
+}
+
+/* ------------------ List Styles ------------------ */
+
+.region-content ul,
+.region-content ol {
+  padding: 2.5em 0 0.25em 0;
+}
+.item-list ul li {
+  padding: 0.2em 0 0 0.5em;
+}
+ul.tips {
+  padding: 0 1.25em 0 0;
+}
+
+/* ------------------ Header ------------------ */
+
+#header #logo {
+  float: right;
+  padding: 15px 0 15px 30px;
+}
+/* Menus when in the header region. */
+#header div.block-menu,
+#header div#block-system-main-menu,
+#header div#block-system-secondary-menu {
+  float: right;
+}
+#header .block-menu li a,
+#header #block-system-main-menu li a,
+#header #block-system-secondary-menu li a {
+  float: right;
+  border-left: 1px solid #555;
+  border-right: none;
+}
+#header .block-menu li.first a,
+#header #block-system-main-menu li.first a,
+#header #block-system-secondary-menu li.first a {
+  padding-right: 0;
+  padding-left: 12px;
+}
+#header .block-menu li.last a,
+#header #block-system-main-menu li.last a,
+#header #block-system-secondary-menu li.last a {
+  padding-left: 0;
+  padding-right: 12px;
+  border-left: none;
+}
+#header #block-search-form {
+  float: left;
+}
+#header #block-user-login {
+  left: 2%;
+  right: auto;
+  float: left;
+}
+#header #block-user-login .form-text {
+  margin-left: 5px;
+}
+#header #block-user-login .item-list ul li {
+  margin: 0 15px 0 5px;
+}
+#header #block-user-login .userbox {
+  margin-right: 10px;
+}
+
+/* --------------- Main Navigation ------------ */
+
+#navigation ul.links li.first {
+  padding-right: 2px;
+  padding-left: 0;
+}
+
+/* ----------------- Content ------------------ */
+
+#block-system-main .submitted .user-picture img {
+  float: right;
+  margin-left: 5px;
+  margin-right: 0;
+}
+#block-system-main div.field-name-taxonomy-tags div.field-label,
+#block-system-main div.field-name-taxonomy-tags div.field-items,
+#block-system-main div.field-name-taxonomy-tags div.field-item {
+  float: right;
+  padding-left: 10px;
+  padding-right: 0;
+}
+#block-system-main .link-wrapper {
+  margin-right: 236px;
+  margin-left: 0;
+}
+
+/* ----------------- Comments ----------------- */
+
+#comment-wrapper div.user-picture img {
+  margin-right: 0;
+}
+#comments .attribution {
+  float: right;
+}
+#comments .comment-arrow {
+  background: url(../images/comment-arrow-rtl.png);
+  margin-right: -45px;
+}
+#comments .comment-text {
+  margin-right: 140px;
+  margin-left: 0;
+}
+#comments .indented {
+  margin-right: 40px;
+  margin-left: 0;
+}
+
+/* -------------- Password Meter  ------------- */
+
+#password-strength  {
+  left: auto;
+  margin-top: 2em;
+  right: 16em;
+}
+#password-strength-text {
+  margin-top: 0;
+  float: left;
+}
+.form-item-pass-pass2 label {
+  clear: right;
+}
+
+/* ------------------ Footer ------------------ */
+
+#footer-columns ul {
+  padding-right: 0;
+}
+#footer-columns li a {
+  padding: 0.8em 20px 0.8em 2px;
+}
+#footer li a {
+  float: right;
+  border-left: 1px solid #555;
+  border-right: none;
+}
+#footer li.first a {
+  padding-right: 0;
+  padding-left: 12px;
+}
+#footer li.last a {
+  padding-left: 0;
+  padding-right: 12px;
+  border-left: none;
+}
+
+/* --------------- System Tabs  --------------- */
+
+.tabs ul.primary li {
+   margin: 0 0 0 5px;
+  float: right;
+  zoom: 1;
+}
+.tabs ul.secondary li {
+  float: right;
+}
+.tabs ul.secondary li:last-child {
+  border-left: none;
+}
+ul.action-links li a  {
+  background-position: right center;
+  padding-left: 0;
+  padding-right: 15px;
+}
+
+/* -------------- Form Elements   ------------- */
+
+.fieldset-legend span.summary {
+  margin-left: 0;
+}
+#user-profile-form input#edit-submit {
+  margin-left: 0;
+}
+.password-suggestions ul li {
+  margin-right: 1.2em;
+  margin-left: 0;
+}
+
+/* Animated throbber */
+html.js input.form-autocomplete {
+  background-position: 1% 4px;
+}
+html.js input.throbbing {
+  background-position: 1% -16px;
+}
+
+/* -------------- Shortcut Links ------------- */
+
+.shortcut-wrapper h1#page-title {
+  float: right;
+}
+
+/* ---------- Poll ----------- */
+
+.poll .vote-form {
+  text-align: right;
+}
+.poll .total {
+  text-align: left;
+}
+
+/* ---------- Color Form ----------- */
+
+.color-form #palette {
+  margin-left: 0;
+  margin-right: 20px;
+}
+.color-form .form-item label {
+  float: right;
+}
+.color-form #palette .lock {
+  right: -20px;
+  left: 0;
+}
diff --git themes/bartik/css/style.css themes/bartik/css/style.css
new file mode 100644
index 0000000..e1c7060
--- /dev/null
+++ themes/bartik/css/style.css
@@ -0,0 +1,1270 @@
+/* $Id$ */
+
+/* ---------- Overall Specifications ---------- */
+
+body,
+#preview {
+  background-color: #fff;
+  line-height: 1.4em;
+  font-size: 0.8em;
+  word-wrap: break-word;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+}
+a:link,
+a:visited {
+  text-decoration: none;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+#preview h1,
+#preview h2,
+#preview h3,
+#preview h4,
+#preview h5,
+#preview h6 {
+  margin: 1.0em 0 0.5em;
+  font-weight: inherit;
+}
+h1,
+#preview h1 {
+  font-size: 140%;
+  color: #000;
+}
+h2,
+#preview h1 {
+  font-size: 120%;
+}
+p,
+#preview p {
+  margin: 0 0 1.8em;
+}
+del {
+  text-decoration: line-through;
+}
+tr.odd {
+  background-color: #dddddd;
+}
+img,
+#preview img {
+  outline: 0;
+}
+
+/* ------------------ Fonts ------------------ */
+
+body,
+#preview,
+#header #site-slogan,
+.ui-widget {
+  font-family: Georgia, "Times New Roman", Times, serif;
+}
+
+#header,
+#footer-wrapper,
+#preview #preview-header,
+ul.contextual-links,
+ul.links,
+ul.primary,
+div.field-type-taxonomy-term-reference,
+div.messages,
+div.meta,
+p.comment-time,
+table {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+input,
+textarea,
+select,
+a.button {
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Verdana, sans-serif;
+}
+
+/* ------------------ Reset Styles ------------------ */
+
+caption {
+  text-align: left; /* LTR */
+  font-weight: normal;
+}
+blockquote{
+  background: #f7f7f7;
+  border-left: 1px solid #bbb;
+  font-style: italic;
+  margin: 1.5em 10px;
+  padding: .5em 10px;
+}
+blockquote:before {
+  color: #bbb;
+  content: "\201C";
+  font-size: 3em;
+  line-height: .1em;
+  margin-right: .2em;
+  vertical-align: -.4em;
+}
+blockquote:after {
+  color: #bbb;
+  content: "\201D";
+  font-size: 3em;
+  line-height: .1em;
+  vertical-align: -.45em;
+}
+blockquote :first-child {
+  display: inline;
+}
+
+a.feed-icon {
+  display: inline-block;
+  padding: 15px 0 0 0;
+}
+
+/* ------------------ Table Styles ------------------ */
+
+table {
+  border-spacing: 0;
+  font-size: 0.85em;
+  width: 100%;
+  margin: 10px 0;
+}
+#footer-wrapper table {
+  font-size: 1em;
+}
+table tr th {
+  background-color: #757575;
+  border-bottom-style: none;
+}
+table thead tr th,
+table thead tr th a,
+table thead tr th a:hover {
+  color: #FFF;
+  font-weight: bold;
+}
+tr td,
+tr th {
+  padding: 4px 9px;
+  border: 1px solid #fff;
+  text-align: left; /* LTR */
+}
+#footer-wrapper tr td,
+#footer-wrapper tr th {
+  border-color: #3b3b3b;
+}
+tr.odd {
+  background-color: #e4e4e4;
+}
+tr,
+tr.even {
+  background-color: #efefef;
+}
+#block-system-main table ul.links {
+  font-size: 1em;
+  margin: 0;
+  padding: 0;
+}
+#block-system-main table ul.links li {
+  padding: 0 1em 0 0;
+}
+
+/* ------------------ List Styles ------------------ */
+
+.block ul {
+  margin: 0;
+}
+ul.contextual-links {
+  font-size: 90%;
+}
+ul.menu li {
+  margin: 0;
+}
+.region-content ul,
+.region-content ol {
+  margin: 1em 0;
+  padding: 0 0 0.25em 2.5em;  /* LTR */
+}
+.item-list ul li {
+  margin: 0;
+  padding: 0.2em 0.5em 0 0;  /* LTR */
+}
+ul.tips {
+  padding: 0 0 0 1.25em;  /* LTR */
+}
+
+/* ------------------ Header ------------------ */
+
+#skip-link {
+  position: absolute;
+  top: -9999px;
+}
+#header,
+#preview #preview-header {
+  color: #fff;
+}
+#header a,
+#preview #preview-header a {
+  color: #fff;
+}
+#header a:hover,
+#header a:focus {
+  color: #b5b7b9;
+}
+#header a:active {
+  color: #adb0bf;
+}
+#header #logo,
+#preview #preview-header #preview-logo {
+  float: left; /* LTR */
+  padding: 15px 30px 15px 0; /* LTR */
+}
+#header #name-and-slogan,
+#preview #preview-header #preview-name-and-slogan {
+  padding-top: 38px;
+  margin-bottom: 30px;
+}
+#header #site-name,
+#preview #preview-header #preview-site-name {
+  font-size: 200%;
+  color: #686868;
+}
+#header h1#site-name,
+#preview #preview-header h1#preview-site-name {
+  margin: 0;
+}
+#header #site-name a,
+#preview-header #preview-site-name a {
+  font-weight: normal;
+}
+#header #site-slogan {
+  margin-top: 7px;
+  font-size: 100%;
+  word-spacing: 0.1em;
+  font-style: italic;
+}
+/* Menus when in the header region. */
+#header div.block-menu,
+#header div#block-system-main-menu,
+#header div#block-system-secondary-menu {
+  height: 1.1em;
+  margin: 10px 0;
+  font-size: 120%;
+  float: left; /* LTR */
+  width: 75%;
+}
+#header .block ul, /* any menu block in the header region */
+#header .block li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+#header .block-menu li a,
+#header #block-system-main-menu li a,
+#header #block-system-secondary-menu li a {
+  float: left; /* LTR */
+  padding: 0 12px;
+  display: block;
+  border-right: 1px solid #555; /* LTR */
+}
+#header .block-menu li.first a,
+#header #block-system-main-menu li.first a,
+#header #block-system-secondary-menu li.first a {
+  padding-left: 0; /* LTR */
+}
+#header .block-menu li.last a,
+#header #block-system-main-menu li.last a,
+#header #block-system-secondary-menu li.last a {
+  padding-right: 0; /* LTR */
+  border-right: none; /* LTR */
+}
+#header #block-search-form { /* search block in the header region */
+  float: right; /* LTR */
+  width: 24%;
+  margin: 8px 0;
+}
+#header #block-user-login { /* login block in the Header Region */
+  display: block;
+  position: absolute;
+  top: 0;
+  right: 2%; /* LTR */
+  background: #464748;
+  color: #fff;
+  float: right; /* LTR */
+  font-size: 13px;
+  padding: 4px 10px 5px;
+  text-transform: uppercase;
+  font-size: 80%;
+}
+#header #block-user-login a {
+  color: #fff;
+  font-size: 90%;
+}
+#header #block-user-login .form-item label {
+  display: inline;
+  font-weight: normal;
+}
+#header #block-user-login .form-item {
+  display: inline;
+  font-weight: normal;
+}
+#header #block-user-login .form-text {
+  margin-right: 5px; /* LTR */
+}
+#header #block-user-login .form-item label .form-required {
+  display: none;
+}
+#header #block-user-login div.item-list,
+#header #block-user-login .item-list ul {
+  display: inline;
+}
+#header #block-user-login .item-list ul li {
+  display: inline;
+  margin: 0 5px 0 15px; /* LTR */
+}
+#header #block-user-login div.item-list {
+  margin-top: 5px;
+  font-style: italic;
+  font-size: 130%;
+  text-transform: none;
+}
+#header #block-user-login .userbox {
+  margin-left: 10px; /* LTR */
+}
+#header #block-user-login input.form-submit {
+  color: #fff;
+  background: #929599;
+  border: 1px solid #73767c;
+  padding: 1px;
+  text-transform: uppercase;
+  margin-top: 1px;
+  position: relative;
+  display: inline;
+}
+#header #block-user-login #user-login-form div.item-list { /* hide create new account + email replacement password links */
+  display: none;
+}
+
+/* --------------- Main Navigation ------------ */
+
+#navigation {
+  padding: 0 15px;
+  clear: both;
+}
+#navigation a {
+  color: #d9d9d9;
+  padding: 0.6em 1em 0.4em;
+}
+#navigation ul {
+  padding: 2px 0;
+}
+#navigation ul.links {
+  font-size: 105%;
+  padding: 0.6em 0.6em 4px;
+}
+#navigation ul.links li a {
+  color: #333;
+  background: #ccc;
+  background: rgba(255, 255, 255, 0.78);
+  border-bottom: 1px solid #fff;
+  text-shadow: 0 1px #eee;
+  -khtml-border-radius-topleft: 8px;
+  -moz-border-radius-topleft: 8px;
+  -webkit-border-top-left-radius: 8px;
+  border-top-left-radius: 8px;
+  -khtml-border-radius-topright: 8px;
+  -moz-border-radius-topright: 8px;
+  -webkit-border-top-right-radius: 8px;
+  border-top-right-radius: 8px;
+}
+#navigation ul.links li a:hover,
+#navigation ul.links li a:focus {
+  background: #fff;
+  background: rgba(255, 255, 255, 0.95);
+}
+#navigation ul.links li a:active {
+  background: #b3b3b3;
+  background: rgba(255, 255, 255, 1);
+}
+#navigation ul.links li.active-trail a {
+  background: #fff;
+  background: rgba(255, 255, 255, 1.0);
+  border-bottom: none;
+}
+.featured #navigation ul.links li.active-trail a {
+  background: #f0f0f0;
+  background: rgba(240, 240, 240, 1.0);
+}
+#navigation ul.links li {
+  display: inline;
+  list-style-type: none;
+  padding: 0.6em 0 0.4em;
+}
+#navigation ul.links li.first {
+  padding-left: 2px; /* LTR */
+}
+
+/* ------------------- Main ------------------- */
+
+#main,
+#preview #preview-main {
+  margin-top: 20px;
+  margin-bottom: 40px;
+}
+
+/* ----------------- Featured ----------------- */
+
+#featured {
+  text-align: center;
+  font-size: 180%;
+  font-weight: normal;
+  line-height: 1.4em;
+  padding: 50px 0 45px;
+  margin: 0;
+  background: #f0f0f0;
+  border-bottom: 1px solid #e7e7e7;
+  text-shadow: 1px 1px #fff;
+}
+#featured p {
+  margin: 0;
+  padding: 0;
+}
+
+/* ----------------- Content ------------------ */
+
+.content {
+  margin-top: 1em;
+}
+#block-system-main,
+#preview #preview-block-system-main {
+  font-size: 115%;
+  line-height: 1.5em;
+}
+body.page-admin-appearance-settings-bartik #block-system-main {
+  font-size: 100%;
+}
+h1#page-title,
+#preview h1#preview-page-title {
+  font-size: 220%;
+  line-height: 1.2em;
+}
+#block-system-main h2,
+#preview #preview-block-system-main h2 {
+  margin-bottom: 2px;
+  font-size: 137%;
+  line-height: 1.4em;
+}
+#block-system-main .node-teaser h2 a {
+  color: #181818;
+}
+#block-system-main .node-teaser {
+  border-bottom: 1px solid #d3d7d9;
+  margin-bottom: 30px;
+  padding-bottom: 15px;
+}
+#block-system-main .node-teaser .content {
+  font-size: 92%;
+  line-height: 1.65em;
+  color: #3b3b3b;
+  clear: none;
+}
+#block-system-main .meta {
+  font-size: 80%;
+  color: #a8a8a8;
+  margin-bottom: -5px;
+}
+* html #block-system-main .meta {
+  margin-bottom: 10px;
+}
+*:first-child+html #block-system-main .meta {
+  margin-bottom: 10px;
+}
+#block-system-main .submitted .user-picture img {
+  float: left; /* LTR */
+  height: 20px;
+  margin: 1px 5px 0 0; /* LTR */
+}
+#block-system-main div.field-type-taxonomy-term-reference div.field-label,
+#block-system-main div.field-type-taxonomy-term-reference div.field-items,
+#block-system-main div.field-type-taxonomy-term-reference div.field-item {
+  display: inline;
+  float: left; /* LTR */
+  padding-right: 10px; /* LTR */
+  font-weight: normal;
+}
+#block-system-main div.field-name-taxonomy-tags div.field-label {
+  font-size: 86%;
+  color: #68696b;
+}
+#block-system-main div.field-name-taxonomy-tags div.field-items,
+#block-system-main div.field-name-taxonomy-tags div.field-item {
+  font-size: 95%;
+}
+#block-system-main div.field-name-taxonomy-tags a:hover,
+#block-system-main div.field-name-taxonomy-tags a:focus {
+  text-decoration: underline;
+}
+#block-system-main .link-wrapper {
+  text-align: center;
+}
+#block-system-main ul.links {
+  color: #68696b;
+  font-size: 80%;
+}
+#block-system-main .field-type-image img,
+#block-system-main .user-picture img {
+  border: 1px solid #d3d7d9;
+  margin: 0;
+}
+
+/* ----------------- Comments ----------------- */
+
+#comment-wrapper {
+  border-top: 1px solid #d3d7d9;
+  padding-top: 15px;
+}
+#comments h2.title {
+  margin-bottom: 1em;
+}
+#comments div.user-picture img {
+  margin-left: 0; /* LTR */
+}
+#comments .comment {
+  margin-bottom: 20px;
+}
+#comments .attribution {
+  float: left; /* LTR */
+  width: 110px;
+}
+#comments .attribution img {
+  margin: 0;
+  border: 1px solid #d3d7d9;
+}
+#comments .submitted p {
+  margin: 4px 0;
+  font-size: 110%;
+  line-height: 1.2em;
+}
+#comments .submitted .comment-time {
+  font-size: 70%;
+  color: #68696b;
+}
+#comments .content {
+  font-size: 90%;
+}
+#comments .comment-arrow {
+  background: url(../images/comment-arrow.png); /* LTR */
+  height: 40px;
+  width: 20px;
+  margin-left: -45px; /* LTR */
+  margin-top: 10px;
+  position: absolute;
+}
+#comments .comment-text {
+  margin-left: 140px; /* LTR */
+  padding: 10px 25px;
+  border: 1px solid #d3d7d9;
+}
+#comments .indented {
+  margin-left: 40px; /* LTR */
+}
+
+/* ------------------ Sidebar ----------------- */
+
+.sidebar .block {
+  border: 1px solid;
+  padding: 15px 20px;
+  margin: 20px 0;
+}
+.sidebar h2,
+#preview .sidebar h2 {
+  margin: 0 0 0.5em;
+  border-bottom: 1px solid #d6d6d6;
+  padding-bottom: 5px;
+  text-shadow: 0 1px 0 #fff;
+}
+.sidebar tbody {
+  border: none;
+}
+.sidebar tr.even,
+.sidebar tr.odd {
+  background: none;
+  border-bottom: 1px solid #d6d6d6;
+}
+
+/* ----------------- Triptych ----------------- */
+
+#triptych-wrapper {
+  background: #fafafa url(../images/triptych-background.jpg) repeat-x top center;
+  border-top: 1px solid #d3d7d9;
+}
+#triptych h2 {
+  color: #000;
+  font-size: 194%;
+  margin-bottom: .8em;
+  text-shadow: 0px 1px 0 #fff;
+  text-align: center;
+  line-height: 1.2em;
+}
+#triptych .block {
+  margin-bottom: 2em;
+  padding-bottom: 2em;
+  border-bottom: 1px solid #dfdfdf;
+}
+#triptych .block.last {
+  border-bottom: none;
+}
+#triptych .block ul li,
+#triptych .block ol li {
+  list-style: none;
+}
+#triptych .block ul,
+#triptych .block ol {
+  padding-left: 0;
+}
+#triptych #block-user-login .form-text {
+  width: 185px;
+}
+#triptych #block-user-online p {
+  margin-bottom: 0;
+}
+#triptych #block-node-syndicate h2 {
+  overflow: hidden;
+  width: 0;
+  height: 0;
+}
+#triptych-last #block-node-syndicate {
+  text-align: right;
+}
+#triptych #block-search-form .form-type-textfield input {
+  width: 185px;
+}
+#triptych-middle #block-system-powered-by {
+  text-align: center;
+}
+#triptych-last #block-system-powered-by {
+  text-align: right;
+}
+
+/* ------------------ Footer ------------------ */
+
+#footer-wrapper {
+  color: #808080;
+  font-size: 90%;
+}
+#footer-wrapper a {
+  color: #a1a1a1;
+}
+#footer-wrapper a:hover,
+#footer-wrapper a:focus {
+  color: #e2e2e2;
+}
+#footer-wrapper div.block {
+  margin: 20px 0;
+  border: 1px solid #38393b;
+  padding: 10px;
+}
+#footer-wrapper #block-system-main-menu,
+#footer-wrapper #block-system-secondary-menu,
+#footer-wrapper .section div.block-menu,
+#footer-wrapper #footer .block {
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+#footer-wrapper #footer .block,
+#footer-wrapper #footer .block .content {
+  overflow: hidden;
+  margin: .5em 0;
+}
+#footer-wrapper #footer .block h2 {
+  margin: 0;
+}
+#footer-wrapper ul#secondary-menu {
+  margin: 1em 0;
+}
+#footer-columns h2 {
+  border-bottom: 1px solid #38393b;
+  font-size: 104%;
+  margin-bottom: 0;
+  padding-bottom: 3px;
+  text-transform: uppercase;
+}
+#footer-columns .content {
+  margin-top: 0;
+}
+#footer-columns p {
+  margin-top: 1em;
+}
+#footer-columns .content ul {
+  list-style: none;
+  padding-left: 0; /* LTR */
+  margin-left: 0
+}
+#footer-columns .content li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+#footer-columns .content li a {
+  display: block;
+  border-bottom: 1px solid #38393b;
+  line-height: 1.2em;
+  padding: 0.8em 2px 0.8em 20px; /* LTR */
+  text-indent: -15px;
+}
+#footer-columns .content li a:hover,
+#footer-columns .content li a:focus {
+  background-color: #1f1f21;
+  background-color: rgba(255,255,255,.05);
+}
+#footer {
+  font-size: 92%;
+  letter-spacing: 0.2px;
+  margin-top: 30px;
+  border-top: 1px solid #333436;
+}
+#footer p {
+  text-align: center;
+}
+#footer .region {
+  margin-top: 20px;
+}
+#footer .block {
+  clear: both;
+}
+#footer ul,
+#footer li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+#footer li a {
+  float: left; /* LTR */
+  padding: 0 12px;
+  display: block;
+  border-right: 1px solid #555; /* LTR */
+}
+#footer li.first a {
+  padding-left: 0; /* LTR */
+}
+#footer li.last a {
+  padding-right: 0; /* LTR */
+  border-right: none; /* LTR */
+}
+#footer-wrapper tr.odd {
+  background-color: transparent;
+}
+#footer-wrapper tr.even {
+  background-color: #1f1f1f;
+}
+
+/* --------------- System Tabs  --------------- */
+
+.tabs {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  margin-bottom: 20px;
+}
+.tabs ul.primary {
+  padding: 0 3px;
+  margin: 0;
+  overflow: hidden;
+  border: none;
+  background: transparent url(../images/tabs-border.png) repeat-x left bottom;
+}
+.tabs ul.primary li {
+  display: block;
+  float: left; /* LTR */
+  vertical-align: bottom;
+  margin: 0 5px 0 0; /* LTR */
+}
+.tabs ul.primary li a.active {
+  border-bottom: 1px solid #fff;
+}
+.tabs ul.primary li a {
+  color: #000;
+  height: 1.8em;
+  line-height: 1.8em;
+  display: block;
+  float: left; /* not LTR */
+  padding: 0 10px 3px;
+  margin: 0;
+  text-shadow: 0 1px 0 #fff;
+  -khtml-border-radius-topleft: 6px;
+  -moz-border-radius-topleft: 6px;
+  -webkit-border-top-left-radius: 6px;
+  border-top-left-radius: 6px;
+  -khtml-border-radius-topright: 6px;
+  -moz-border-radius-topright: 6px;
+  -webkit-border-top-right-radius: 6px;
+  border-top-right-radius: 6px;
+}
+.tabs ul.secondary {
+  border-bottom: none;
+}
+.tabs ul.secondary li {
+  display: block;
+  float: left; /* LTR */
+}
+.tabs ul.secondary li:last-child {
+  border-right: none; /* LTR */
+}
+.tabs ul.secondary li a {
+  padding: .25em .5em;
+}
+.tabs ul.secondary li a.active {
+  background: #f2f2f2;
+  border-bottom: none;
+  -moz-border-radius: 5px;
+  -webkit-border-radius: 5px;
+  -khtml-border-radius: 5px;
+  border-radius: 5px;
+}
+ul.action-links {
+  list-style: none;
+  margin: 5px;
+  padding: .5em 1em;
+}
+ul.action-links li {
+  display: inline-block;
+  margin-left: 10px;
+}
+ul.action-links li a {
+  padding-left: 15px;
+  background: url(../images/add.png) no-repeat left center;
+  margin: 0 10px 0 0;
+}
+
+/* ---------------- Messages  ----------------- */
+
+#messages {
+  padding: 20px 0 5px;
+  margin: 0 auto;
+}
+div.messages {
+  padding: 1.2em 2em 1em;
+  margin: 8px 0;
+}
+div.status, tr.status {
+  background-color: #c7ffc0;
+  border: 1px solid #89d47f;
+}
+div.warning, tr.warning {
+  background-color: #fcfca7;
+  border: 1px solid #e1c46b;
+}
+div.error, tr.error {
+  background-color: #ffcccc;
+  border: 1px solid #fb6b6b;
+}
+
+/* -------------- User Profile   -------------- */
+
+.profile .user-picture {
+  float: none;
+}
+
+/* -------------- Password Meter  ------------- */
+
+.password-parent,
+div.form-item div.password-suggestions {
+  position: relative;
+  width: auto;
+}
+#password-strength {
+  float: none;
+  left: 16em;
+  position: absolute;
+  width: 11.5em;
+}
+#password-strength-text,
+.password-strength-title,
+div.password-confirm {
+  font-size: 0.82em;
+}
+#password-strength-text {
+  margin-top: 0.2em;
+}
+
+/* ---------------- Buttons    ---------------- */
+
+input.form-submit,
+a.button {
+  background: #fff url(../images/buttons.png) 0 0 repeat-x;
+  border: 1px solid #e4e4e4;
+  border-bottom: 1px solid #b4b4b4;
+  border-left-color: #d2d2d2;
+  border-right-color: #d2d2d2;
+  color: #3a3a3a;
+  cursor: pointer;
+  font-size: 90%;
+  font-weight: normal;
+  text-align: center;
+  margin-bottom: 1em;
+  padding: 4px 17px;
+  -khtml-border-radius: 15px;
+  -moz-border-radius: 20px;
+  -webkit-border-radius: 15px;
+  border-radius: 15px;
+}
+a.button:link,
+a.button:visited,
+a.button:hover,
+a.button:focus,
+a.button:active {
+  text-decoration: none;
+  color: #5a5a5a;
+}
+
+/* -------------- Form Elements   ------------- */
+
+fieldset {
+  background: #ffffff;
+  border: 1px solid #cccccc;
+  margin-top: 10px;
+  margin-bottom: 20px;
+  padding: 0 0 10px;
+  position: relative;
+  -khtml-border-radius: 4px;
+  -moz-border-radius: 4px;
+  -webkit-border-radius: 4px;
+  border-radius: 4px;
+}
+.fieldset-wrapper {
+  margin-top: 25px;
+}
+.filter-wrapper {
+  -khtml-border-radius-topright: 0;
+  -khtml-border-radius-topleft: 0;
+  -moz-border-radius-topright: 0;
+  -moz-border-radius-topleft: 0;
+  -webkit-border-top-left-radius: 0;
+  -webkit-border-top-right-radius: 0;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+}
+fieldset.collapsed {
+  background: transparent;
+  -khtml-border-radius: 0;
+  -moz-border-radius: 0;
+  -webkit-border-radius: 0;
+  border-radius: 0;
+}
+fieldset legend {
+  background: #dbdbdb;
+  border: 1px solid #ccc;
+  border-bottom: none;
+  color: #3b3b3b;
+  display: block;
+  height: 2em;
+  left: -1px; /* LTR */
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Verdana, sans-serif;
+  line-height: 2em;
+  padding: 0;
+  position: absolute;
+  text-indent: 10px;
+  text-shadow: 0 1px 0 #fff;
+  top: -12px;
+  width: 100%;
+  -khtml-border-radius-topleft: 4px;
+  -moz-border-radius-topleft: 4px;
+  -webkit-border-top-left-radius: 4px;
+  border-top-left-radius: 4px;
+  -khtml-border-radius-topright: 4px;
+  -moz-border-radius-topright: 4px;
+  -webkit-border-top-right-radius: 4px;
+  border-top-right-radius: 4px;
+}
+fieldset.collapsed legend {
+  -khtml-border-radius: 4px;
+  -moz-border-radius: 4px;
+  -webkit-border-radius: 4px;
+  border-radius: 4px;
+}
+fieldset legend a {
+  color: #3b3b3b;
+}
+fieldset legend a:hover,
+fieldset legend a:focus,
+fieldset legend a:active {
+  color: #000;
+}
+fieldset .fieldset-wrapper {
+  padding: 0 10px;
+}
+input {
+  margin: 2px 0;
+  padding: 4px;
+}
+textarea.form-textarea,
+select.form-select {
+  padding: 4px;
+}
+input.form-text,
+textarea.form-textarea,
+select.form-select {
+  border: 1px solid #ccc;
+}
+input.form-submit:hover,
+input.form-submit:focus {
+  background: #dedede;
+}
+.password-suggestions ul li {
+  margin-left: 1.2em; /* LTR */
+}
+.form-item {
+  margin-bottom: 1em;
+  margin-top: 2px;
+}
+.form-item label {
+  font-size: 90%;
+}
+.form-radio {
+  margin-right: 4px;
+}
+fieldset .description {
+  margin-top: 5px;
+  line-height: 1.4em;
+  color: #3c3c3c;
+  font-style: italic;
+}
+.form-actions {
+  padding-top: 10px;
+}
+
+/* Animated throbber */
+html.js input.form-autocomplete {
+  background-position: 100% 4px; /* LTR */
+}
+html.js input.throbbing {
+  background-position: 100% -16px; /* LTR */
+}
+
+/* -------------- Other Overrides ------------- */
+
+div.password-suggestions {
+  border: 0;
+}
+.ui-widget-overlay {
+  background: #222222;
+  opacity: 0.7;
+}
+div.vertical-tabs .vertical-tabs-panes fieldset.vertical-tabs-pane {
+  padding: 1em;
+}
+
+/* --------------- Search Form ---------------- */
+
+#block-search-form .content {
+  margin-top: 0;
+}
+#block-search-form .form-item-search-block-form input {
+  width: 70%;
+}
+.region-content #block-search-form .form-item-search-block-form input,
+.region-footer #block-search-form .form-item-search-block-form input {
+  width: auto;
+}
+#block-search-form #edit-actions {
+  float: right;
+}
+.region-content #block-search-form #edit-actions,
+.region-footer #block-search-form #edit-actions {
+  float: none;
+}
+#block-search-form .form-actions {
+  padding-top: 0;
+}
+#search-block-form input.form-submit,
+#search-form input.form-submit {
+  height: 24px;
+  width: 14px;
+  overflow: hidden;
+  cursor: pointer;
+  text-indent: -9999px;
+  border: none;
+  background: url(../images/search-button.png) no-repeat left center;
+}
+
+/* -------------- Shortcut Links -------------- */
+
+.shortcut-wrapper {
+  margin: 2.2em 0 1.1em 0; /* Same as usual h1#page-title margin. */
+}
+.shortcut-wrapper h1#page-title {
+  float: left; /* LTR */
+  margin: 0;
+}
+div.add-or-remove-shortcuts {
+  padding-top: 0.9em;
+}
+.overlay div.add-or-remove-shortcuts {
+  padding-top: 0.8em;
+}
+
+/* ---------- Admin-specific Theming ---------- */
+
+.page-admin #block-system-main img {
+  margin-right: 15px; /* LTR */
+}
+.page-admin-structure-block-demo .block-region {
+ color: #000000;
+}
+.page-admin #admin-dblog img {
+  margin: 0 5px;
+}
+/* Fix spacing when Seven is used in the overlay. */
+#system-theme-settings fieldset {
+  padding: 0;
+}
+#system-theme-settings fieldset .fieldset-legend {
+  margin-top: 0;
+}
+
+/* Configuration */
+div.admin .right,
+div.admin .left {
+  width: 49%;
+  margin: 0;
+}
+div.admin-panel {
+  background: #fbfbfb;
+  border: 1px solid #ccc;
+  margin: 10px 0;
+  padding: 0px 5px 5px;
+}
+div.admin-panel h3 {
+  margin: 16px 7px;
+}
+div.admin-panel dt {
+  border-top: 1px solid #ccc;
+  padding: 7px 0 0;
+}
+div.admin-panel dd {
+  margin: 0 0 10px;
+}
+div.admin-panel .description {
+  margin: 0 0 14px 7px;
+}
+
+/* ---------- Overlay layout styles ----------- */
+
+.overlay #main,
+.overlay #content {
+  width: auto;
+  float: none;
+}
+.overlay #page {
+  padding: 0 2em;
+}
+.overlay #skip-link,
+.overlay .region-page-top,
+.overlay #header,
+.overlay #page-title,
+.overlay #featured,
+.overlay #sidebar-first,
+.overlay #triptych-wrapper,
+.overlay #footer-wrapper {
+  display: none;
+}
+.overlay-processed .field-type-image {
+ display: block;
+ float: none;
+}
+.overlay #messages {
+  width: auto;
+}
+
+/* ---------- Poll ----------- */
+.node .poll {
+  margin: 2em 0;
+}
+.node .poll #edit-choice {
+  margin: 0 0 1.5em;
+}
+.poll .vote-form {
+  text-align: left; /* LTR */
+}
+.poll .vote-form .choices {
+  margin: 0;
+}
+.poll .percent {
+  font-size: 86%;
+  font-style: italic;
+  margin-bottom: 3em;
+  margin-top: -3.2em;
+  float: right;
+  text-align: right;
+}
+.poll .text {
+  clear: right;
+}
+.poll .total {
+  font-size: 95%;
+  font-style: italic;
+  text-align: right; /* LTR */
+  clear: both;
+}
+.poll .form-item label {
+  font-size: 100%;
+}
+.node .poll {
+  margin: 1.8em 0 0;
+}
+.node .poll #edit-choice {
+  margin: 0 0 1.2em;
+}
+.poll .bar .foreground {
+  background-color: #666;
+}
+#footer-wrapper .poll .bar {
+  background-color: #666;
+}
+#footer-wrapper .poll .bar .foreground {
+  background-color: #ddd;
+}
+
+/* ---------- Color Form ----------- */
+
+.color-form #placeholder {
+  position: static;
+  width: 195px;
+}
+.overlay .color-form #placeholder {
+  position: absolute;
+}
+.color-form #palette {
+  margin-left: 20px;  /* LTR */
+}
+.color-form .form-item {
+  border: 1px solid #fbfbfb;
+  margin: 0;
+  padding: .5em;
+  position: relative;
+  max-width: 350px;
+  -khtml-border-radius: 5px;
+  -moz-border-radius: 5px;
+  -webkit-border-radius: 5px;
+  border-radius: 5px;
+}
+.color-form .form-item label {
+  float: left; /* LTR */
+  font-size: 1em;
+  width: 150px;
+}
+.color-form .form-item.item-selected {
+  background-color: #eeeeee;
+  border: 1px solid #cfcfcf;
+}
+.color-form #palette .lock {
+  position: absolute;
+  top: 5px;
+  left: -20px; /* LTR */
+}
+.color-form #preview {
+  font-size: .85em;
+}
+.overlay #preview #footer-wrapper {
+  display: block;
+}
diff --git themes/bartik/scripts/search.js themes/bartik/scripts/search.js
new file mode 100644
index 0000000..c97fc8e
--- /dev/null
+++ themes/bartik/scripts/search.js
@@ -0,0 +1,26 @@
+// $Id$
+(function ($) {
+
+  Drupal.behaviors.bartik = {
+    attach: function(context) {
+
+      $('#search-block-form input[type=text]').val(Drupal.t('Search'));
+
+      $('#search-block-form input[type=text]').focus(function(){
+        $(this).addClass('has-focus');
+        if ($(this).val() == Drupal.t('Search')) {
+          $(this).val('');
+        }
+      });
+
+      $('#search-block-form input[type=text]').blur(function(){
+        $(this).removeClass('has-focus');
+        if ($(this).val() == '') {
+          $(this).val(Drupal.t('Search'));
+        }
+      });
+    }
+
+  };
+
+})(jQuery);
diff --git themes/bartik/template.php themes/bartik/template.php
new file mode 100644
index 0000000..e4257b1
--- /dev/null
+++ themes/bartik/template.php
@@ -0,0 +1,138 @@
+<?php
+// $Id$
+
+/**
+ * Add body classes if certain regions have content.
+ */
+function bartik_preprocess_html(&$variables) {
+  if (!empty($variables['page']['featured'])) {
+    $variables['classes_array'][] = 'featured';
+  }
+
+  if (!empty($variables['page']['triptych_first'])
+    || !empty($variables['page']['triptych_middle'])
+    || !empty($variables['page']['triptych_last'])) {
+    $variables['classes_array'][] = 'triptych';
+  }
+
+  if (!empty($variables['page']['footer_firstcolumn'])
+    || !empty($variables['page']['footer_secondcolumn'])
+    || !empty($variables['page']['footer_thirdcolumn'])
+    || !empty($variables['page']['footer_fourthcolumn'])) {
+    $variables['classes_array'][] = 'footer-columns';
+  }
+
+  // Add conditional stylesheets for IE
+  drupal_add_css(path_to_theme() . '/css/ie.css', array('weight' => CSS_THEME, 'browsers' => array('IE' => 'lte IE 7', '!IE' => FALSE), 'preprocess' => FALSE));
+  drupal_add_css(path_to_theme() . '/css/ie6.css', array('weight' => CSS_THEME, 'browsers' => array('IE' => 'IE 6', '!IE' => FALSE), 'preprocess' => FALSE));
+}
+
+/**
+ * Override or insert variables into the page template for HTML output.
+ */
+function bartik_process_html(&$variables) {
+  // Hook into color.module.
+  if (module_exists('color')) {
+    _color_html_alter($variables);
+  }
+}
+
+/**
+ * Override or insert variables into the page template.
+ */
+function bartik_process_page(&$variables) {
+  // Hook into color.module.
+  if (module_exists('color')) {
+    _color_page_alter($variables);
+  }
+  // Always print the site name, but don't always show it visually.
+  $variables['hide_site_name']   = theme_get_setting('toggle_name') ? FALSE : TRUE;
+  $variables['hide_site_slogan'] = theme_get_setting('toggle_slogan') ? FALSE : TRUE;
+  if ($variables['hide_site_name']) {
+    // If toggle_name is FALSE, the site_name will be empty, so we rebuild it.
+    $variables['site_name'] = filter_xss_admin(variable_get('site_name', 'Drupal'));
+  }
+  if ($variables['hide_site_slogan']) {
+    // If toggle_site_slogan is FALSE, the site_slogan will be empty, so we rebuild it.
+    $variables['site_slogan'] = filter_xss_admin(variable_get('site_slogan', ''));
+  }
+  // Since the title and the shortcut link are both block level elements,
+  // positioning them next to each other is much simpler with a wrapper div.
+  if (!empty($variables['title_suffix']['add_or_remove_shortcut']) && $variables['title']) {
+    // Add a wrapper div using the title_prefix and title_suffix render elements.
+    $variables['title_prefix']['shortcut_wrapper'] = array(
+      '#markup' => '<div class="shortcut-wrapper clearfix">',
+      '#weight' => 100,
+    );
+    $variables['title_suffix']['shortcut_wrapper'] = array(
+      '#markup' => '</div>',
+      '#weight' => -99,
+    );
+    // Make sure the shortcut link is the first item in title_suffix.
+    $variables['title_suffix']['add_or_remove_shortcut']['#weight'] = -100;
+  }
+}
+
+/**
+ * Override or insert variables into the maintenance page template.
+ */
+function bartik_process_maintenance_page(&$variables) {
+  // Always print the site name, but don't always show it visually.
+  $variables['hide_site_name']   = theme_get_setting('toggle_name') ? FALSE : TRUE;
+  $variables['hide_site_slogan'] = theme_get_setting('toggle_slogan') ? FALSE : TRUE;
+  if ($variables['hide_site_name']) {
+    // If toggle_name is FALSE, the site_name will be empty, so we rebuild it.
+    $variables['site_name'] = filter_xss_admin(variable_get('site_name', 'Drupal'));
+  }
+  if ($variables['hide_site_slogan']) {
+    // If toggle_site_slogan is FALSE, the site_slogan will be empty, so we rebuild it.
+    $variables['site_slogan'] = filter_xss_admin(variable_get('site_slogan', ''));
+  }
+}
+
+/**
+ * Override or insert variables into the block template.
+ */
+function bartik_preprocess_block(&$variables) {
+  // In the header region, visually hide the title of any menu block or of the
+  // user login block, but leave it accessible.
+  if ($variables['block']->region == 'header' && ($variables['block']->module == 'menu' || $variables['block']->module == 'user' && $variables['block']->delta == 'login')) {
+    $variables['title_attributes_array']['class'][] = 'element-invisible';
+  }
+  // Set "first" and "last" classes.
+  if ($variables['block']->position_first){
+    $variables['classes_array'][] = 'first';
+  }
+  if ($variables['block']->position_last){
+    $variables['classes_array'][] = 'last';
+  }
+  // Set "odd" & "even" classes.
+  $variables['classes_array'][] = $variables['block']->position%2 == 0 ? 'even' : 'odd';
+}
+
+/**
+ * Implements hook_page_alter.
+ */
+function bartik_page_alter(&$page) {
+  // Determine the position and count of blocks within regions.
+  foreach ($page as &$region) {
+    // Make sure this is a "region" element.
+    if (is_array($region) && isset($region['#region'])) {
+      $i = 0;
+      foreach ($region as &$block) {
+        // Make sure this is a "block" element.
+        if (is_array($block) && isset($block['#block'])) {
+          $block['#block']->position = $i++;
+        }
+      }
+      $region['#block_count'] = $i++;
+      // Set a flag for "first" and "last" blocks.
+      foreach ($region as &$block) {
+        if (is_array($block) && isset($block['#block']->position)) {
+          $block['#block']->position_first = $block['#block']->position == 0;
+          $block['#block']->position_last = $block['#block']->position == $region['#block_count']-1;
+        }
+      }
+    }
+  }
+}
diff --git themes/bartik/templates/comment-wrapper.tpl.php themes/bartik/templates/comment-wrapper.tpl.php
new file mode 100644
index 0000000..66e9e79
--- /dev/null
+++ themes/bartik/templates/comment-wrapper.tpl.php
@@ -0,0 +1,54 @@
+<?php
+// $Id$
+
+/**
+ * @file
+ * Bartik's theme implementation to provide an HTML container for comments.
+ *
+ * Available variables:
+ * - $content: The array of content-related elements for the node. Use
+ *   render($content) to print them all, or
+ *   print a subset such as render($content['comment_form']).
+ * - $classes: String of classes that can be used to style contextually through
+ *   CSS. It can be manipulated through the variable $classes_array from
+ *   preprocess functions. The default value has the following:
+ *   - comment-wrapper: The current template type, i.e., "theming hook".
+ * - $title_prefix (array): An array containing additional output populated by
+ *   modules, intended to be displayed in front of the main title tag that
+ *   appears in the template.
+ * - $title_suffix (array): An array containing additional output populated by
+ *   modules, intended to be displayed after the main title tag that appears in
+ *   the template.
+ *
+ * The following variables are provided for contextual information.
+ * - $node: Node object the comments are attached to.
+ * The constants below the variables show the possible values and should be
+ * used for comparison.
+ * - $display_mode
+ *   - COMMENT_MODE_FLAT
+ *   - COMMENT_MODE_THREADED
+ *
+ * Other variables:
+ * - $classes_array: Array of html class attribute values. It is flattened
+ *   into a string within the variable $classes.
+ *
+ * @see template_preprocess_comment_wrapper()
+ * @see theme_comment_wrapper()
+ */
+?>
+<div id="comments-wrapper">
+  <div id="comments" class="<?php print $classes; ?>"<?php print $attributes; ?>>
+    <?php if ($content['comments'] && $node->type != 'forum'): ?>
+      <?php print render($title_prefix); ?>
+      <h2 class="title"><?php print t('Comments'); ?></h2>
+      <?php print render($title_suffix); ?>
+    <?php endif; ?>
+
+    <?php print render($content['comments']); ?>
+
+    <?php if ($content['comment_form']): ?>
+      <h2 class="title comment-form"><?php print t('Add new comment'); ?></h2>
+      <?php print render($content['comment_form']); ?>
+    <?php endif; ?>
+  </div>
+</div>
diff --git themes/bartik/templates/comment.tpl.php themes/bartik/templates/comment.tpl.php
new file mode 100644
index 0000000..07555da
--- /dev/null
+++ themes/bartik/templates/comment.tpl.php
@@ -0,0 +1,102 @@
+<?php
+// $Id$
+
+/**
+ * @file
+ * Bartik's theme implementation for comments.
+ *
+ * Available variables:
+ * - $author: Comment author. Can be link or plain text.
+ * - $content: An array of comment items. Use render($content) to print them all, or
+ *   print a subset such as render($content['field_example']). Use
+ *   hide($content['field_example']) to temporarily suppress the printing of a
+ *   given element.
+ * - $created: Formatted date and time for when the comment was created.
+ *   Preprocess functions can reformat it by calling format_date() with the
+ *   desired parameters on the $comment->created variable.
+ * - $changed: Formatted date and time for when the comment was last changed.
+ *   Preprocess functions can reformat it by calling format_date() with the
+ *   desired parameters on the $comment->changed variable.
+ * - $new: New comment marker.
+ * - $permalink: Comment permalink.
+ * - $picture: Authors picture.
+ * - $signature: Authors signature.
+ * - $status: Comment status. Possible values are:
+ *   comment-unpublished, comment-published or comment-preview.
+ * - $title: Linked title.
+ * - $classes: String of classes that can be used to style contextually through
+ *   CSS. It can be manipulated through the variable $classes_array from
+ *   preprocess functions. The default values can be one or more of the following:
+ *   - comment: The current template type, i.e., "theming hook".
+ *   - comment-by-anonymous: Comment by an unregistered user.
+ *   - comment-by-node-author: Comment by the author of the parent node.
+ *   - comment-preview: When previewing a new or edited comment.
+ *   The following applies only to viewers who are registered users:
+ *   - comment-unpublished: An unpublished comment visible only to administrators.
+ *   - comment-by-viewer: Comment by the user currently viewing the page.
+ *   - comment-new: New comment since last the visit.
+ * - $title_prefix (array): An array containing additional output populated by
+ *   modules, intended to be displayed in front of the main title tag that
+ *   appears in the template.
+ * - $title_suffix (array): An array containing additional output populated by
+ *   modules, intended to be displayed after the main title tag that appears in
+ *   the template.
+ *
+ * These two variables are provided for context:
+ * - $comment: Full comment object.
+ * - $node: Node object the comments are attached to.
+ *
+ * Other variables:
+ * - $classes_array: Array of html class attribute values. It is flattened
+ *   into a string within the variable $classes.
+ *
+ * @see template_preprocess()
+ * @see template_preprocess_comment()
+ * @see template_process()
+ * @see theme_comment()
+ */
+?>
+<div class="<?php print $classes; ?> clearfix"<?php print $attributes; ?>>
+
+  <div class="attribution">
+
+    <?php print $picture; ?>
+
+    <div class="submitted">
+      <?php print $permalink; ?>
+      <p class="commenter-name">
+        <?php print $author; ?>
+      </p>
+      <p class="comment-time">
+        <?php print $created; ?>
+      </p>
+    </div>
+  </div>
+
+  <div class="comment-text">
+    <div class="comment-arrow"></div>
+
+    <?php if ($new): ?>
+      <span class="new"><?php print $new; ?></span>
+    <?php endif; ?>
+
+    <?php print render($title_prefix); ?>
+    <h3<?php print $title_attributes; ?>><?php print $title; ?></h3>
+    <?php print render($title_suffix); ?>
+
+    <div class="content"<?php print $content_attributes; ?>>
+      <?php
+        // We hide the comments and links now so that we can render them later.
+        hide($content['links']);
+        print render($content);
+      ?>
+      <?php if ($signature): ?>
+      <div class="user-signature clearfix">
+        <?php print $signature; ?>
+      </div>
+      <?php endif; ?>
+    </div> <!-- /.content -->
+
+    <?php print render($content['links']); ?>
+  </div> <!-- /.comment-text -->
+</div>
diff --git themes/bartik/templates/maintenance-page.tpl.php themes/bartik/templates/maintenance-page.tpl.php
new file mode 100644
index 0000000..2103b10
--- /dev/null
+++ themes/bartik/templates/maintenance-page.tpl.php
@@ -0,0 +1,174 @@
+<?php
+// $Id$
+
+/**
+ * @file
+ * Bartik's theme implementation to display a single Drupal page while offline.
+ *
+ * All the available variables are mirrored in page.tpl.php. Some may be left
+ * blank but they are provided for consistency.
+ *
+ * @see template_preprocess()
+ * @see template_preprocess_maintenance_page()
+ * @see bartik_process_maintenance_page()
+ */
+?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
+  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php print $language->language; ?>" lang="<?php print $language->language; ?>" dir="<?php print $language->dir; ?>">
+
+<head>
+  <?php print $head; ?>
+  <title><?php print $head_title; ?></title>
+  <?php print $styles; ?>
+  <?php print $scripts; ?>
+</head>
+<body class="<?php print $classes; ?>" <?php print $attributes;?>>
+  <div id="skip-link">
+    <a href="#main-content"><?php print t('Skip to main content'); ?></a>
+  </div>
+  <?php print $page_top; ?>
+
+<div id="page-wrapper"><div id="page">
+
+  <div id="header"><div class="section clearfix">
+
+    <?php if ($logo): ?>
+      <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" id="logo">
+        <img src="<?php print $logo; ?>" alt="<?php print t('Home'); ?>" />
+      </a>
+    <?php endif; ?>
+
+    <?php if ($site_name || $site_slogan): ?>
+      <div id="name-and-slogan"<?php if ($hide_site_name && $hide_site_slogan) { print ' class="element-invisible"'; } ?>>
+
+        <?php if ($site_name): ?>
+          <?php if ($title): ?>
+            <div id="site-name"<?php if ($hide_site_name) { print ' class="element-invisible"'; } ?>>
+              <strong>
+                <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+              </strong>
+            </div>
+          <?php else: /* Use h1 when the content title is empty */ ?>
+            <h1 id="site-name"<?php if ($hide_site_name) { print ' class="element-invisible"'; } ?>>
+              <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+            </h1>
+          <?php endif; ?>
+        <?php endif; ?>
+
+        <?php if ($site_slogan): ?>
+          <div id="site-slogan"<?php if ($hide_site_slogan) { print ' class="element-invisible"'; } ?>>
+            <?php print $site_slogan; ?>
+          </div>
+        <?php endif; ?>
+
+      </div> <!-- /#name-and-slogan -->
+    <?php endif; ?>
+
+    <?php print $header; ?>
+
+  </div></div> <!-- /.section, /#header -->
+
+  <?php if ($messages): ?>
+    <div id="messages"><div class="section clearfix">
+      <?php print $messages; ?>
+    </div></div> <!-- /.section, /#messages -->
+  <?php endif; ?>
+
+  <?php if ($featured): ?>
+    <div id="featured" class="region"><div class="section clearfix">
+      <?php print $featured; ?>
+    </div></div> <!-- /.section, /#featured -->
+  <?php endif; ?>
+
+  <div id="main-wrapper"><div id="main" class="clearfix">
+
+    <div id="content" class="column"><div class="section">
+      <?php if ($highlight): ?><div id="highlight"><?php print $highlight; ?></div><?php endif; ?>
+      <a id="main-content"></a>
+      <?php if ($title): ?>
+        <h1 class="title" id="page-title">
+          <?php print $title; ?>
+        </h1>
+      <?php endif; ?>
+      <?php print $content; ?>
+
+    </div></div> <!-- /.section, /#content -->
+
+    <?php if ($sidebar_first): ?>
+      <div id="sidebar-first" class="column sidebar"><div class="section">
+        <?php print $sidebar_first; ?>
+      </div></div> <!-- /.section, /#sidebar-first -->
+    <?php endif; ?>
+
+  </div></div> <!-- /#main, /#main-wrapper -->
+
+  <?php if ($triptych_first || $triptych_middle || $triptych_last): ?>
+    <div id="triptych-wrapper"><div id="triptych" class="clearfix">
+
+      <?php if ($triptych_first): ?>
+        <div id="triptych-first" class="region triptych"><div class="section">
+          <?php print $triptych_first; ?>
+        </div></div> <!-- /.section, /#triptych-first -->
+      <?php endif; ?>
+
+      <?php if ($triptych_middle): ?>
+        <div id="triptych-middle" class="region triptych"><div class="section">
+          <?php print $triptych_middle; ?>
+        </div></div> <!-- /.section, /#triptych-middle -->
+      <?php endif; ?>
+
+      <?php if ($triptych_last): ?>
+        <div id="triptych-last" class="region triptych"><div class="section">
+          <?php print $triptych_last; ?>
+        </div></div> <!-- /.section, /#triptych-last -->
+      <?php endif; ?>
+
+    </div></div> <!-- /#triptych, /#triptych-wrapper -->
+  <?php endif; ?>
+
+  <div id="footer-wrapper"><div class="section">
+
+    <?php if ($footer_firstcolumn || $footer_secondcolumn || $footer_thirdcolumn || $footer_fourthcolumn): ?>
+      <div id="footer-columns" class="clearfix">
+
+        <?php if ($footer_firstcolumn): ?>
+          <div id="footer-firstcolumn" class="region sitemap"><div class="section">
+            <?php print $footer_firstcolumn; ?>
+          </div></div> <!-- /.section, /#footer-firstcolumn -->
+        <?php endif; ?>
+
+        <?php if ($footer_secondcolumn): ?>
+          <div id="footer-secondcolumn" class="region sitemap"><div class="section">
+            <?php print $footer_secondcolumn; ?>
+          </div></div> <!-- /.section, /#footer-secondcolumn -->
+        <?php endif; ?>
+
+        <?php if ($footer_thirdcolumn): ?>
+          <div id="footer-thirdcolumn" class="region sitemap"><div class="section">
+            <?php print $footer_thirdcolumn; ?>
+          </div></div> <!-- /.section, /#footer-thirdcolumn -->
+        <?php endif; ?>
+
+        <?php if ($footer_fourthcolumn): ?>
+          <div id="footer-fourthcolumn" class="region sitemap"><div class="section">
+            <?php print $footer_fourthcolumn; ?>
+          </div></div> <!-- /.section, /#footer-fourthcolumn -->
+        <?php endif; ?>
+
+      </div><!-- /#footer-columns -->
+    <?php endif; ?>
+
+    <?php if ($footer): ?>
+      <div id="footer" class="clearfix">
+        <?php print $footer; ?>
+      </div><!-- /#footer -->
+    <?php endif; ?>
+
+  </div></div> <!-- /.section, /#footer-wrapper -->
+
+</div></div> <!-- /#page, /#page-wrapper -->
+
+  <?php print $page_bottom; ?>
+</body>
+</html>
diff --git themes/bartik/templates/node.tpl.php themes/bartik/templates/node.tpl.php
new file mode 100644
index 0000000..e3ada69
--- /dev/null
+++ themes/bartik/templates/node.tpl.php
@@ -0,0 +1,126 @@
+<?php
+// $Id$
+
+/**
+ * @file
+ * Bartik's theme implementation to display a node.
+ *
+ * Available variables:
+ * - $title: the (sanitized) title of the node.
+ * - $content: An array of node items. Use render($content) to print them all,
+ *   or print a subset such as render($content['field_example']). Use
+ *   hide($content['field_example']) to temporarily suppress the printing of a
+ *   given element.
+ * - $user_picture: The node author's picture from user-picture.tpl.php.
+ * - $date: Formatted creation date. Preprocess functions can reformat it by
+ *   calling format_date() with the desired parameters on the $created variable.
+ * - $name: Themed username of node author output from theme_username().
+ * - $node_url: Direct url of the current node.
+ * - $display_submitted: whether submission information should be displayed.
+ * - $classes: String of classes that can be used to style contextually through
+ *   CSS. It can be manipulated through the variable $classes_array from
+ *   preprocess functions. The default values can be one or more of the
+ *   following:
+ *   - node: The current template type, i.e., "theming hook".
+ *   - node-[type]: The current node type. For example, if the node is a
+ *     "Blog entry" it would result in "node-blog". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - node-teaser: Nodes in teaser form.
+ *   - node-preview: Nodes in preview mode.
+ *   The following are controlled through the node publishing options.
+ *   - node-promoted: Nodes promoted to the front page.
+ *   - node-sticky: Nodes ordered above other non-sticky nodes in teaser
+ *     listings.
+ *   - node-unpublished: Unpublished nodes visible only to administrators.
+ * - $title_prefix (array): An array containing additional output populated by
+ *   modules, intended to be displayed in front of the main title tag that
+ *   appears in the template.
+ * - $title_suffix (array): An array containing additional output populated by
+ *   modules, intended to be displayed after the main title tag that appears in
+ *   the template.
+ *
+ * Other variables:
+ * - $node: Full node object. Contains data that may not be safe.
+ * - $type: Node type, i.e. story, page, blog, etc.
+ * - $comment_count: Number of comments attached to the node.
+ * - $uid: User ID of the node author.
+ * - $created: Time the node was published formatted in Unix timestamp.
+ * - $classes_array: Array of html class attribute values. It is flattened
+ *   into a string within the variable $classes.
+ * - $zebra: Outputs either "even" or "odd". Useful for zebra striping in
+ *   teaser listings.
+ * - $id: Position of the node. Increments each time it's output.
+ *
+ * Node status variables:
+ * - $view_mode: View mode, e.g. 'full', 'teaser'...
+ * - $teaser: Flag for the teaser state (shortcut for $view_mode == 'teaser').
+ * - $page: Flag for the full page state.
+ * - $promote: Flag for front page promotion state.
+ * - $sticky: Flags for sticky post setting.
+ * - $status: Flag for published status.
+ * - $comment: State of comment settings for the node.
+ * - $readmore: Flags true if the teaser content of the node cannot hold the
+ *   main body content.
+ * - $is_front: Flags true when presented in the front page.
+ * - $logged_in: Flags true when the current user is a logged-in member.
+ * - $is_admin: Flags true when the current user is an administrator.
+ *
+ * Field variables: for each field instance attached to the node a corresponding
+ * variable is defined, e.g. $node->body becomes $body. When needing to access
+ * a field's raw values, developers/themers are strongly encouraged to use these
+ * variables. Otherwise they will have to explicitly specify the desired field
+ * language, e.g. $node->body['en'], thus overriding any language negotiation
+ * rule that was previously applied.
+ *
+ * @see template_preprocess()
+ * @see template_preprocess_node()
+ * @see template_process()
+ */
+?>
+<div id="node-<?php print $node->nid; ?>" class="<?php print $classes; ?> clearfix"<?php print $attributes; ?>>
+
+  <?php print render($title_prefix); ?>
+  <?php if (!$page): ?>
+    <h2<?php print $title_attributes; ?>>
+      <a href="<?php print $node_url; ?>"><?php print $title; ?></a>
+    </h2>
+  <?php endif; ?>
+  <?php print render($title_suffix); ?>
+
+  <?php if ($display_submitted): ?>
+    <div class="meta submitted">
+      <?php print $user_picture; ?>
+      <?php
+        print t('published by !username on !datetime',
+          array('!username' => $name, '!datetime' => $date));
+      ?>
+    </div>
+  <?php endif; ?>
+
+  <div class="content clearfix"<?php print $content_attributes; ?>>
+    <?php
+      // We hide the comments and links now so that we can render them later.
+      hide($content['comments']);
+      hide($content['links']);
+      print render($content);
+    ?>
+  </div>
+
+  <?php
+    // Remove the "Add new comment" link on the teaser page or if the comment
+    // form is being displayed on the same page.
+    if ($teaser || !empty($content['comments']['comment_form'])) {
+      unset($content['links']['comment']['#links']['comment-add']);
+    }
+    // Only display the wrapper div if there are links.
+    $links = render($content['links']);
+    if ($links):
+  ?>
+    <div class="link-wrapper">
+      <?php print $links; ?>
+    </div>
+  <?php endif; ?>
+
+  <?php print render($content['comments']); ?>
+
+</div>
diff --git themes/bartik/templates/page.tpl.php themes/bartik/templates/page.tpl.php
new file mode 100644
index 0000000..87f6a73
--- /dev/null
+++ themes/bartik/templates/page.tpl.php
@@ -0,0 +1,280 @@
+<?php
+// $Id$
+
+/**
+ * @file
+ * Bartik's theme implementation to display a single Drupal page.
+ *
+ * The doctype, html, head and body tags are not in this template. Instead they
+ * can be found in the html.tpl.php template normally located in the
+ * modules/system folder.
+ *
+ * Available variables:
+ *
+ * General utility variables:
+ * - $base_path: The base URL path of the Drupal installation. At the very
+ *   least, this will always default to /.
+ * - $directory: The directory the template is located in, e.g. modules/system
+ *   or themes/garland.
+ * - $is_front: TRUE if the current page is the front page.
+ * - $logged_in: TRUE if the user is registered and signed in.
+ * - $is_admin: TRUE if the user has permission to access administration pages.
+ *
+ * Site identity:
+ * - $front_page: The URL of the front page. Use this instead of $base_path,
+ *   when linking to the front page. This includes the language domain or
+ *   prefix.
+ * - $logo: The path to the logo image, as defined in theme configuration.
+ * - $site_name: The name of the site, empty when display has been disabled
+ *   in theme settings.
+ * - $site_slogan: The slogan of the site, empty when display has been disabled
+ *   in theme settings.
+ * - $hide_site_name: TRUE if the site name has been toggled off on the theme
+ *   settings page. If hidden, the "element-invisible" class is added to make
+ *   the site name visually hidden, but still accessible.
+ * - $hide_site_slogan: TRUE if the site slogan has been toggled off on the
+ *   theme settings page. If hidden, the "element-invisible" class is added to
+ *   make the site slogan visually hidden, but still accessible.
+ *
+ * Navigation:
+ * - $main_menu (array): An array containing the Main menu links for the
+ *   site, if they have been configured.
+ * - $secondary_menu (array): An array containing the Secondary menu links for
+ *   the site, if they have been configured.
+ * - $breadcrumb: The breadcrumb trail for the current page.
+ *
+ * Page content (in order of occurrence in the default page.tpl.php):
+ * - $title_prefix (array): An array containing additional output populated by
+ *   modules, intended to be displayed in front of the main title tag that
+ *   appears in the template.
+ * - $title: The page title, for use in the actual HTML content.
+ * - $title_suffix (array): An array containing additional output populated by
+ *   modules, intended to be displayed after the main title tag that appears in
+ *   the template.
+ * - $messages: HTML for status and error messages. Should be displayed
+ *   prominently.
+ * - $tabs (array): Tabs linking to any sub-pages beneath the current page
+ *   (e.g., the view and edit tabs when displaying a node).
+ * - $action_links (array): Actions local to the page, such as 'Add menu' on the
+ *   menu administration interface.
+ * - $feed_icons: A string of all feed icons for the current page.
+ * - $node: The node object, if there is an automatically-loaded node
+ *   associated with the page, and the node ID is the second argument
+ *   in the page's path (e.g. node/12345 and node/12345/revisions, but not
+ *   comment/reply/12345).
+ *
+ * Regions:
+ * - $page['header']: Items for the header region.
+ * - $page['featured']: Items for the featured region.
+ * - $page['highlight']: Items for the highlighted content region.
+ * - $page['help']: Dynamic help text, mostly for admin pages.
+ * - $page['content']: The main content of the current page.
+ * - $page['sidebar_first']: Items for the first sidebar.
+ * - $page['triptych_first']: Items for the first triptych.
+ * - $page['triptych_middle']: Items for the middle triptych.
+ * - $page['triptych_last']: Items for the last triptych.
+ * - $page['footer_firstcolumn']: Items for the first footer column.
+ * - $page['footer_secondcolumn']: Items for the second footer column.
+ * - $page['footer_thirdcolumn']: Items for the third footer column.
+ * - $page['footer_fourthcolumn']: Items for the fourth footer column.
+ * - $page['footer']: Items for the footer region.
+ *
+ * @see template_preprocess()
+ * @see template_preprocess_page()
+ * @see template_process()
+ * @see bartik_process_page()
+ */
+?>
+<div id="page-wrapper"><div id="page">
+
+  <div id="header"><div class="section clearfix">
+
+    <?php if ($logo): ?>
+      <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" id="logo">
+        <img src="<?php print $logo; ?>" alt="<?php print t('Home'); ?>" />
+      </a>
+    <?php endif; ?>
+
+    <?php if ($site_name || $site_slogan): ?>
+      <div id="name-and-slogan"<?php if ($hide_site_name && $hide_site_slogan) { print ' class="element-invisible"'; } ?>>
+
+        <?php if ($site_name): ?>
+          <?php if ($title): ?>
+            <div id="site-name"<?php if ($hide_site_name) { print ' class="element-invisible"'; } ?>>
+              <strong>
+                <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+              </strong>
+            </div>
+          <?php else: /* Use h1 when the content title is empty */ ?>
+            <h1 id="site-name"<?php if ($hide_site_name) { print ' class="element-invisible"'; } ?>>
+              <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+            </h1>
+          <?php endif; ?>
+        <?php endif; ?>
+
+        <?php if ($site_slogan): ?>
+          <div id="site-slogan"<?php if ($hide_site_slogan) { print ' class="element-invisible"'; } ?>>
+            <?php print $site_slogan; ?>
+          </div>
+        <?php endif; ?>
+
+      </div> <!-- /#name-and-slogan -->
+    <?php endif; ?>
+
+    <?php print render($page['header']); ?>
+
+    <?php if ($main_menu): ?>
+      <div id="navigation"><div class="section clearfix">
+        <?php print theme('links__system_main_menu', array(
+          'links' => $main_menu,
+          'attributes' => array(
+            'id' => 'main-menu',
+            'class' => array('links', 'clearfix'),
+          ),
+          'heading' => array(
+            'text' => t('Main menu'),
+            'level' => 'h2',
+            'class' => array('element-invisible'),
+          ),
+        )); ?>
+      </div></div> <!-- /.section, /#navigation -->
+    <?php endif; ?>
+
+  </div></div> <!-- /.section, /#header -->
+
+  <?php if ($messages): ?>
+    <div id="messages"><div class="section clearfix">
+      <?php print $messages; ?>
+    </div></div> <!-- /.section, /#messages -->
+  <?php endif; ?>
+
+  <?php if ($page['featured']): ?>
+    <div id="featured" class="region"><div class="section clearfix">
+      <?php print render($page['featured']); ?>
+    </div></div> <!-- /.section, /#featured -->
+  <?php endif; ?>
+
+  <div id="main-wrapper" class="clearfix"><div id="main" class="clearfix">
+
+    <?php if ($breadcrumb): ?>
+      <div id="breadcrumb"><?php print $breadcrumb; ?></div>
+    <?php endif; ?>
+
+    <?php if ($page['sidebar_first']): ?>
+      <div id="sidebar-first" class="column sidebar"><div class="section">
+        <?php print render($page['sidebar_first']); ?>
+      </div></div> <!-- /.section, /#sidebar-first -->
+    <?php endif; ?>
+
+
+    <div id="content" class="column"><div class="section">
+      <?php if ($page['highlight']): ?><div id="highlight"><?php print render($page['highlight']); ?></div><?php endif; ?>
+      <a id="main-content"></a>
+      <?php print render($title_prefix); ?>
+      <?php if ($title): ?>
+        <h1 class="title" id="page-title">
+          <?php print $title; ?>
+        </h1>
+      <?php endif; ?>
+      <?php print render($title_suffix); ?>
+      <?php if ($tabs): ?>
+        <div class="tabs">
+          <?php print render($tabs); ?>
+        </div>
+      <?php endif; ?>
+      <?php print render($page['help']); ?>
+      <?php if ($action_links): ?>
+        <ul class="action-links">
+          <?php print render($action_links); ?>
+        </ul>
+      <?php endif; ?>
+      <?php print render($page['content']); ?>
+      <?php print $feed_icons; ?>
+
+    </div></div> <!-- /.section, /#content -->
+
+    <?php if ($page['sidebar_second']): ?>
+      <div id="sidebar-second" class="column sidebar"><div class="section">
+        <?php print render($page['sidebar_second']); ?>
+      </div></div> <!-- /.section, /#sidebar-second -->
+    <?php endif; ?>
+
+  </div></div> <!-- /#main, /#main-wrapper -->
+
+  <?php if ($page['triptych_first'] || $page['triptych_middle'] || $page['triptych_last']): ?>
+    <div id="triptych-wrapper"><div id="triptych" class="clearfix">
+
+      <?php if ($page['triptych_first']): ?>
+        <div id="triptych-first" class="region triptych"><div class="section">
+          <?php print render($page['triptych_first']); ?>
+        </div></div> <!-- /.section, /#triptych-first -->
+      <?php endif; ?>
+
+      <?php if ($page['triptych_middle']): ?>
+        <div id="triptych-middle" class="region triptych"><div class="section">
+          <?php print render($page['triptych_middle']); ?>
+        </div></div> <!-- /.section, /#triptych-middle -->
+      <?php endif; ?>
+
+      <?php if ($page['triptych_last']): ?>
+        <div id="triptych-last" class="region triptych"><div class="section">
+          <?php print render($page['triptych_last']); ?>
+        </div></div> <!-- /.section, /#triptych-last -->
+      <?php endif; ?>
+
+    </div></div> <!-- /#triptych, /#triptych-wrapper -->
+  <?php endif; ?>
+
+  <div id="footer-wrapper"><div class="section">
+
+    <?php if ($page['footer_firstcolumn'] || $page['footer_secondcolumn'] || $page['footer_thirdcolumn'] || $page['footer_fourthcolumn']): ?>
+      <div id="footer-columns" class="clearfix">
+
+        <?php if ($page['footer_firstcolumn']): ?>
+          <div id="footer-firstcolumn" class="region sitemap"><div class="section">
+            <?php print render($page['footer_firstcolumn']); ?>
+          </div></div> <!-- /.section, /#footer-firstcolumn -->
+        <?php endif; ?>
+
+        <?php if ($page['footer_secondcolumn']): ?>
+          <div id="footer-secondcolumn" class="region sitemap"><div class="section">
+            <?php print render($page['footer_secondcolumn']); ?>
+          </div></div> <!-- /.section, /#footer-secondcolumn -->
+        <?php endif; ?>
+
+        <?php if ($page['footer_thirdcolumn']): ?>
+          <div id="footer-thirdcolumn" class="region sitemap"><div class="section">
+            <?php print render($page['footer_thirdcolumn']); ?>
+          </div></div> <!-- /.section, /#footer-thirdcolumn -->
+        <?php endif; ?>
+
+        <?php if ($page['footer_fourthcolumn']): ?>
+          <div id="footer-fourthcolumn" class="region sitemap"><div class="section">
+            <?php print render($page['footer_fourthcolumn']); ?>
+          </div></div> <!-- /.section, /#footer-fourthcolumn -->
+        <?php endif; ?>
+
+      </div><!-- /#footer-columns -->
+    <?php endif; ?>
+
+    <?php if ($page['footer'] || $secondary_menu): ?>
+      <div id="footer" class="clearfix">
+        <?php print theme('links__system_secondary_menu', array(
+          'links' => $secondary_menu,
+          'attributes' => array(
+            'id' => 'secondary-menu',
+            'class' => array('links', 'clearfix'),
+          ),
+          'heading' => array(
+            'text' => t('Secondary menu'),
+            'level' => 'h2',
+            'class' => array('element-invisible'),
+          ),
+        )); ?>
+        <?php print render($page['footer']); ?>
+    </div><!-- /#footer -->
+    <?php endif; ?>
+
+  </div></div> <!-- /.section, /#footer-wrapper -->
+
+</div></div> <!-- /#page, /#page-wrapper -->
