Index: date_api_elements.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/date/date_api_elements.inc,v
retrieving revision 1.66
diff -u -r1.66 date_api_elements.inc
--- date_api_elements.inc	13 Apr 2010 20:10:07 -0000	1.66
+++ date_api_elements.inc	19 Apr 2010 07:26:58 -0000
@@ -152,8 +152,9 @@
   $element['date']['#weight'] = !empty($element['date']['#weight']) ? $element['date']['#weight'] : $element['#weight'];
   $element['date']['#default_value'] = is_object($date) ? date_format_date($date , 'custom', $element['#date_format']) : '';
   $element['date']['#attributes'] = array('class' => isset($element['#attributes']['class']) ? $element['#attributes']['class'] += array('date-date') : array('date-date'));
-  
-  $element['date']['#description'] = ' ' . t('Format: @date', array('@date' => date_format_date(date_make_date(date($element['#date_format'], REQUEST_TIME)), 'custom', $element['#date_format']);
+
+  $formatted_date = date_format_date(date_make_date(date($element['#date_format'], REQUEST_TIME)), 'custom', $element['#date_format']);
+  $element['date']['#description'] = ' ' . t('Format: @date', array('@date' => $formatted_date));
   
   // Keep the system from creating an error message for the sub-element.
   // We'll set our own message on the parent element.
